st2api/st2api/controllers/v1/actionalias.py

```
@@ -58,6 +73,31 @@ def get_all(self, **kwargs):
     def get_one(self, ref_or_id):
         return super(ActionAliasController, self)._get_one(ref_or_id)
 
+    @jsexpose(arg_types=[str], status_code=http_client.ACCEPTED)
+    def match(self, command, **kwargs):
+        """
+            Run a chatops command
+
+            Handles requests:
+                POST /chatops/match
+
+                command=hello%20world
+        """
+        try:
+            # 1. Get aliases
+            aliases = super(ActionAliasController, self)._get_all(**kwargs)
+            # 2. Match alias(es) to command
+            matches = match_command_to_alias(command, aliases)
+            if len(matches) > 1:
+                raise ActionAliasAmbiguityException("Command matched more than 1 pattern",
+                                                    matches=matches)
+            return [self._match_tuple_to_dict(match) for match in matches]
+        except (ActionAliasAmbiguityException) as e:
+            # TODO : error on unmatched alias
```

We should also throw the same exception here as above, right?